spec/collection-spec.coffee

```diff
@@ -88,24 +103,24 @@ describe 'Brainstem.Collection', ->
 
     context 'collection has model without a brainstemKey defined', ->
       beforeEach ->
-        collection.model = new Backbone.Model()
+        collection.model = Backbone.Model
 
       it 'throws a "BrainstemError"', ->
         expect(-> collection.fetch()).toThrow()
 
     context 'the collection has brainstemKey defined', ->
       beforeEach ->
-        collection.model = App.Models.Post
+        collection.model = Post
 
       it 'does not throw', ->
         expect(-> collection.fetch()).not.toThrow()
 
       it 'assigns its BrainstemKey to the options object', ->
-        loadObjectSpy = spyOn(base.data, 'loadObject').andReturn(new $.Deferred)
+        loadObjectSpy = spyOn(storageManager, 'loadObject').and.returnValue(new $.Deferred)
```

Comment:
We can also clean up these deferred objects to not use `new` since it's a factory object and not a constructor. Another item to do in the future.